analyzers/rspec/cs/S3168_c#.html

```diff
@@ -47,6 +47,7 @@ <h2>Compliant Solution</h2>
 }
 </pre>
 <h2>Exceptions</h2>
-<p>Event handlers, i.e. methods with two arguments, first one matching <code>object sender</code> and the second being or inheriting from
-<code>EventArgs</code>, are ignored.</p>
+<p>Event handlers, i.e. methods with two arguments, first one matching type <code>object</code> or name <code>sender</code> and the second being or
+inheriting from <code>EventArgs</code>, are ignored.</p>
+<p>Event handlers named as <code>OnSomething</code> are also ignored.</p>
```

Comment:
I would say "Methods named as `OnSomething` ...", right? We don't know for sure they're event handlers?